src/DeclarativeLayers.ts

```diff
@@ -63,7 +64,7 @@ export class DeclarativeLayers {
     }
     private shouldBeVisibleInitially = (layerMetadata: dataTypes.ILayerMetadata): boolean => {
         // if tagged as visible AND initialized
-       return layerMetadata.visibleInitially && !!this.layerReferences[layerMetadata.id];
+       return defaultToTrue(layerMetadata.visibleInitially) && !!this.layerReferences[layerMetadata.id];
```

Comment:
If the behaviour you want is

- truthy or `undefined` = `true`
- any other falsy value = `false`

you could also do this:
```
      const { visibleInitially = true } = layerMetadata;
      return visibleInitially && !!this.layerReferences[layerMetadata.id];
```

Or remove the extra variable altogether:

```
    private shouldBeVisibleInitially = ({ id, visibleInitially = true}: layerMetadata: dataTypes.ILayerMetadata): boolean => {
        // if tagged as visible AND initialized
      return visibleInitially && !!this.layerReferences[id];
    }
```
Love your declarative style. 👏

Reply:
I dont think that will capture `null` and default it to `true` which Justin is suggesting and I think makes sense to do, BUT, I didnt realize I can default while declaring a variable. That is incredibly useful to know! :)